src/Pscid/Options.purs

```diff
@@ -72,16 +73,50 @@ mkDefaultOptions =
       <*> mkCommand "test"
       <*> scanDefaultDirectories
 
-mkCommand ∷ ∀ e. String → Eff (fs ∷ FS | e) String
+type IncludePath = String
+
+data CLICommand
+  = NamedScriptSpecificCommand String
+  | NamedScriptCommand String
```

Comment:
Since the two NPM commands are treated the same everywhere I'd prefer only two variants here, maybe `PulpCommand` and `ScriptCommand`?

Reply:
Done.